openedx/core/djangolib/markup.py

```diff
@@ -14,6 +15,47 @@
 Text = markupsafe.escape                        # pylint: disable=invalid-name
 
 
+class HTMLCleaner(Cleaner):
+    """
+    HTMLCleaner extends lxml.html.clean.Cleaner to sanitize HTML content while preserving valid URLs
+    and removing unsafe JavaScript links.
+
+    Attributes:
+    -----------
+    _is_url : Callable[[str], Optional[re.Match]]
+        A regular expression pattern used to identify valid URLs. This pattern matches strings that
+        start with 'http', 'https', 'ftp', or 'file' schemes, case-insensitively.
+    """
+    def _remove_javascript_link(self, link: str):
+        """
+        Checks if the given link is a valid URL. If it is, the link is returned unchanged.
```

Comment:
Open edx supports [Google style docstrings](https://docs.openedx.org/en/latest/developers/references/developer_guide/style_guides/python-guidelines.html#docstrings). So I think we should follow that guilde.